session.go

```diff
@@ -1248,6 +1260,14 @@ func (q *Query) reset() {
 	*q = Query{}
 }
 
+var (
```

Comment:
This should live close to IsLWT

Reply:
But it's used here rather than at the other place.

Reply:
I mean this whole block and the func below